easybuild/framework/easyconfig/parser.py

```diff
@@ -106,6 +107,18 @@ def process(self, filename=None):
         self._read(filename=filename)
         self._set_formatter()
 
+    def check_values_types(self, cfg):
+        """Check types of easyconfig parameter values."""
+        wrong_type_msgs = []
+        for key in cfg:
+            if not check_type_of_param_value(key, cfg[key]):
+                wrong_type_msgs.append("value for '%s' should be of type '%s'" % (key, TYPES[key].__name__))
+
+        if wrong_type_msgs:
+            raise EasyBuildError("Type checking of easyconfig parameter values failed: %s", ', '.join(wrong_type_msgs))
+        else:
+            self.log.info("Type checking of easyconfig parameter values passed!")
```

Comment:
This is just a single log line, it's worth having this at the `info` level to indicate type checking was performed.